### PR TITLE
Feat/path customer rights

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**
# Description
What are changes related to ?

# Linked Issues link to #571 
What are the issues fixed by this pull request ? i put add customer right

# Changes
What does it change ?
Is is a breaking change ?
Is is a new feature ?
Is is a patch, fix, hotfix ?

# Screenshots
Put screenshots (if any)

# Tests
How has it been tested ?

- [X] Manually
- [ ] Automated tests
- [ ] QA
- [ ] Other

# Additional information
Precise any other information

### CODE REVIEW

File: yaki_admin_backend/src/test/java/com/xpeho/yaki_admin_backend/data/services/CustomerServiceImplTest.java [hunk 1]
```diff
@@ -125,16 +125,35 @@ void getCustomersTest() {
...
-    void addCustomersRight() {
-        //when
-        List<UserModel> users = List.of(
-                new UserModel());
-        given(customerJpaRepository.getReferenceById(1)).willReturn(customer1);
-        given(customerJpaRepository.save(any(CustomerModel.class))).willReturn(customer1);
-        CustomerEntity customersReturned = customerService.addCustomerRight(users, 1);
-        assertEquals(customersReturned, customerE1);
-    }
+    //@Test
```
Comment (LaurentMag) [inline yaki_admin_backend/src/test/java/com/xpeho/yaki_admin_backend/data/services/CustomerServiceImplTest.java:128]: please remove commented code

File: yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/presentation/controllers/CustomerController.java [hunk 2]
```diff
@@ -31,21 +32,28 @@ public CustomerEntity createCustomer(
         return customerService.createCustomer(customerEntity);
     }
 
+    @PostMapping("/addUser")
```
Comment (LaurentMag) [inline yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/presentation/controllers/CustomerController.java:35]: No camel case for REST routes,
addUser isn't clear. Maybe use something like "/add-customer-right" ?

**Review by LaurentMag — CHANGES_REQUESTED** [review]

File: yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/data/models/UserModel.java [hunk 3]
```diff
@@ -165,4 +164,9 @@ public List<CustomerModel> getCustomers() {
     public void setCustomers(List<CustomerModel> customers) {
         this.customers = customers;
     }
+
+    public int getId() {
```
Comment (PiotrFLEURY) [inline yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/data/models/UserModel.java:168]:
```suggestion
    public int getUserId() {
```

File: yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/presentation/controllers/CustomerController.java [hunk 4]
```diff
@@ -31,21 +32,28 @@ public CustomerEntity createCustomer(
         return customerService.createCustomer(customerEntity);
     }
 
+    @PostMapping("/add-customer-right")
```
Comment (PiotrFLEURY) [inline yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/presentation/controllers/CustomerController.java:35]:
```suggestion
    @PostMapping("/rights")
```

**Review by PiotrFLEURY — COMMENTED** [review]
Some minor changes to apply

File: yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/data/models/UserModel.java (hunk 3 above)
Comment (PriscilliaAmmeux) [reply yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/data/models/UserModel.java:168]: I delete this getter because it already exists

**Review by LaurentMag — APPROVED** [review]

**Review by PiotrFLEURY — APPROVED** [review]
LGTM

**Pull request merged**